static/sass/homepage.scss

```diff
@@ -1,9 +1,12 @@
 $color-navigation-background: #e95420;
 
 @import 'vanilla-framework/scss/vanilla';
+@import 'patterns/_pattern_footer';
+@import 'patterns/_pattern_lists';
 @include vanilla;
+@include ubuntu-p-footer;
+@include ubuntu-p-lists;
```

Comment:
@steverydz Seems like a lot of custom sass just for the footer, and it's also causing CI to fail. Have you tried using Vanilla patterns to replicate the footer? The one on snapcraft.io might be a good starting point.

Also the ubuntu.com sass is kiiiiind of a tangled mess. The less we see of it around, the better imo